schema/property/raw.json

```diff
@@ -23,6 +23,31 @@
         },
         "source": {
             "$ref": "source.json"
+        },
+        "exabyteId": {
```

Comment:
We will need to clean up https://github.com/Exabyte-io/esse/blob/172188ec99a625170ab0770ab066364e5c9fe078/schema/property/refined.json#L11 and remove the obsolete entries then or update - i.e. refined property is a raw property with `isRefined` tag